Gemfile

```diff
@@ -8,6 +8,8 @@ eval_gemfile "Gemfile.devtools"
 
 gemspec
 
+gem "dry-configurable", git: "https://github.com/dry-rb/dry-configurable.git", branch: "master"
```

Comment:
Hey @timriley , is this new line supposed to be here, or is it a leak from your debugging session? :slightly_smiling_face:

Reply:
@waiting-for-dev Thanks for checking :) It's needed here for the time being, to allow the tests on the master branch to pass. Without this they'll fail because they don't work with the latest release of the dry-configurable gem.

When it comes to preparing all these changes for release, I'll do it in the right order (e.g. dry-configurable first, others afterwards) and pull out these Gemfile entries before each release.